### PR TITLE
Final presentation

### PR DESCRIPTION
**Pull request by gilberthappi**
Project Requierements - React App ( Crypto Watcher )
Choose API : [Coin Stats](https://documenter.getpostman.com/view/5734027/RzZ6Hzr3?version=latest) ✅
Design : [Guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) ✅
Interactions
Home page ✅
Details page ✅
Testing requirements
Create unit tests for pure functions in your Redux code. ✅
Create integration tests for your application using the React Testing Library. ✅
Technical requirements
The project is a single-page application (SPA) built with React and Redux. ✅
The data retrieved from the API should be stored in the Redux store. ✅
You should filter the data that you retrieve from the API using a Filter stateless component. ✅
Every page (the main page and the pages for each item) should have a unique route within the SPA. ✅
The project should be deployed and accessible online using [Render](https://render.com/), [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/). ✅
Project documentation
Loom-Project-Demo:  https://www.loom.com/share/5e26e6f940af40de80cedee1ea2b916c?sid=0c7fa8f7-3b9e-4c8b-9054-595ca1b92c2c 
Description of the project. ✅
Live Demo of the project. ✅ https://crypto-watcher-9z4a.onrender.com/
You should also highlight some interesting piece of code or something you built that you are very proud of. ✅
All linters have been checked! ✅

### CODE REVIEW

**Comment by gilberthappi** [comment]
Live Demo of the project. ✅{**[https://crypto-watcher-9z4a.onrender.com/**}](https://crypto-watcher-9z4a.onrender.com/)

File: src/App.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ function App() {
           <Route index element={<Coins />} />
           <Route path="coin/:id" element={<SpecificCoin />} />
           {' '}
+
```
Comment (iambenkis) [inline src/App.js]: - [ ] Keep doing the good work 🚀

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @gilberthappi 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 

- All tests are passing :heavy_check_mark: 
- Use of Routers :heavy_check_mark: 
- Design sticks to the given design :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Please, let's try to keep in mind to be attentive to details. It's a skill that would make you stand out in your future job. You should have a video with  a maximum length of 5 min. It is best to use [loom](https://www.loom.com/) as this helps you not to record more than 5 mins. Please kindly update the video, so it meets the project requirements.

- You should present  the features of the project. 
- The video should contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video should contain a demo of the project, highlighting all the project features.
- The video should contain a highlight about a piece of the code the student finds interesting or is proud of making.

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @gilberthappi 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
 - [x] You have written a clean code ✅
 - [x] Functrional app. ✅

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**